easylist_adult/adult_thirdparty.txt

```diff
@@ -39,6 +39,8 @@
 ||bannershotlink.perfectgonzo.com^
 ||bans.bride.ru^
 ||bate.cum-shows.net^
+||bngdyn.com/promo.php
```

Comment:
You can block these completely as well, I haven't found any frontpage or any other use of them

`||$domain^`